static/src/js/containers/MetricsEventsContainer/__tests__/MetricsEventContainer.test.js

```diff
@@ -71,4 +71,21 @@ describe('MetricsEventsContainer component', () => {
       })
     })
   })
+
+  test('when a title can be found on the elements props', () => {
```

Comment:
Did you mean cannot be found?

Reply:
No but, reading it again this name is bad `when a title can be found on other element props`? Kind of a weird phrase but, its more clear